site/content/blogs/release-v131.ja.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'AWS Copilot v1.31: NLB 設定の拡張, タスク失敗時のログの改善, `copilot deploy` の機能拡張'
+twitter_title: 'AWS Copilot v1.31'
+image: ''
+image_alt: ''
+image_width: '1051'
+image_height: '747'
+---
+
+# AWS Copilot v1.31: NLB 設定の拡張, タスク失敗時のログの改善, `copilot deploy` の機能拡張
```

Comment:
Sorry for nitpicking: please use Japanese comma "、" here
cf) https://github.com/aws/copilot-cli/blob/mainline/site/content/blogs/release-v130.ja.md?plain=1#L10

Reply:
thanks, will fix it.